src/training_database_reader.py

```diff
@@ -0,0 +1,58 @@
+""" training_database_reader.py """
+import os
+import sys
+import sqlite3
+import pandas as pd
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '../')))
+
+class DatabaseReader:
+    """
+    Reads from SQLite database, and creates a pandas dataframe
+    """
+    def __init__(self, database, connection=None, table="TrainingData", query=None):
+        """
+        Initialize DatabaseReader and loads data into a Pandas DataFrame. 
+        
+        Arguements:
```

Comment:
Very helpful commenting. Nit picky but Arguments is spelled wrong (extra e after the u)